### DISCUSSION TITLE
Want to be able to load entire scenery packs into game through object selector

### DISCUSSION THREAD

**Discussion by jamtraks** [OpenRCT2/OpenRCT2#16909] [Feature requests]
Will there ever come a time when i will be able to import complete work bench object sets into my object selection list while playing a saved park which previously did not have those objects or object/workbench sets? Or into any new park etc...
I want to be able to load workbench sets from the object selection menu, i want to select entire workbench sets with one click. Right now i have to select each object manually.
also, the "download missing object files" function doesn't seem to function properly.

**Comment by spacek531** [comment]
Entire workbench sets, as in scenery groups? That already exists, although it may be buggy. If you find a scenery group which fails to load a scenery you have installed, please report the scenery group and scenery item in issues.
